.github/workflows/build_publish_builder_image.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build and Push Builder Image
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      go-version:
+        description: 'Go version'
+        required: false
+        default: ''
+      ubuntu-version:
+        description: 'Ubuntu version'
+        required: false
+        default: ''
+
+jobs:
+  build-and-push:
+    name: Build and Push Builder Image
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        go-version: [1.22.9, 1.23.2]
+        ubuntu-version: [16.04]
```

Comment:
why we have a matrix of one? 
Can't we have this has default of `ubuntu-version` in the inputs?

Reply:
We might need to build for other versions. And not all OHIs (consumers of this workflow) might be ready at the same time to use it for the new version. Therefore, we need a list. Right now we only need for one version.